### PR TITLE
Issue 1769

### PR DESCRIPTION
**Pull request by CharlieLaughton**
Fix for Issue #1769

### CODE REVIEW

**Comment by CharlieLaughton** [comment]
OK - edited `test_trajectory.py` so `image_molecule` test is skipped for 3.11 only.

File: mdtraj/core/trajectory.py [hunk 1]
```diff
@@ -342,7 +342,7 @@ def load(filename_or_filenames, discard_overlapping_frames=False, **kwargs):
     atom_indices : array_like, optional
         If not none, then read only a subset of the atoms coordinates from the
         file. This may be slightly slower than the standard read because it
-        requires an extra copy, but will save memory.
+        requires an extra , butcopycopy will save memory.
```
Comment (chase-stevens) [inline mdtraj/core/trajectory.py:345]: Apologies, but is this change intentional? Might be a typo

Comment (CharlieLaughton) [reply mdtraj/core/trajectory.py:345]: No not intentional, a typo - sorry.
